### ISSUE TITLE
fluxFloat does not return valid float when the value is a number without a decimal part

### ISSUE THREAD

**Issue by mickdekkers** [influxdata/influxdb-client-js#467]
__Expected Behavior:__

```ts
import assert from 'assert';
import { fluxFloat } from '@influxdata/influxdb-client';

const result = fluxFloat(60);

assert(result.toString() === '60.0');
```

__Current Behavior:__

```ts
import assert from 'assert';
import { fluxFloat } from '@influxdata/influxdb-client';

const result = fluxFloat(60);

// Missing .0, not seen as float by InfluxDB
assert(result.toString() === '60');
```

__Possible Solution:__

Make `sanitizeFloat` add the `.0` for numbers without a decimal part even when `typeof value === 'number'`
https://github.com/influxdata/influxdb-client-js/blob/2dbd1c1f1b148e4ea88ade8972efe2aee9cdc277/packages/core/src/query/flux.ts#L114-L120

__Steps/Code to Reproduce the Problem:__

See current behavior.

__Specifications:__

- Version: 1.27.0
- Platform: Node.js

**Comment by sranka** [comment]
Thank you @mickdekkers , I am on it shortly

**Comment by sranka** [comment]
The suggested solution would create a regression in a [flux](https://influxdata.github.io/influxdb-client-js/influxdb-client.flux.html) tagged template, where the sanitizeFloat function sanitize number values, I have to be careful with a solution. `fluxFloat` has to better return `float(v: "${sanitizeString(value)})`, it will also handle numbers with an exponential notation that currently also fail. Additionally +INF, -INF, -0 even NAN will be then properly supported. 

A workaround is to use ```flux`float(v: "${v}")` ```

**Comment by mickdekkers** [comment]
Thanks a lot for the quick fix! Great work @sranka! 😄

**Comment by sranka** [comment]
Sure, it is already fixed in `@influxdata/influxdb-client@1.28.0-nightly.2123`
